core/common/buffer_or_view.hpp

```diff
@@ -15,7 +15,7 @@
 namespace kagome::common {
   /// Moved owned buffer or readonly view.
   class BufferOrView {
-    using Span = gsl::span<const uint8_t>;
+    using Span = BufferView;
```

Comment:
`span` here, not `BufferView`
```suggestion
    using Span = std::span<const uint8_t>;
```